### PR TITLE
Allow movable tabs

### LINKED ISSUE #450: "Help" tab not always on the side
I think it makes sense to have "Help" tab to be always on the side (preferably right, but left okay too). When I open Ginga, open "Help", and then open another channel, "Help" is sandwiched between the two channels, which appears to be an awkward position. I am also unable to manually drag it to the side, so it is stuck there.

Screenshot:
untitled

Note: If I manually close "Help" and then re-open, then it is on the right side again.

### PR DESCRIPTION
**Pull request by pllim**
Fix #450 , as suggested by @ejeschke 

Also general PEP8 clean-up for `Desktop.py`

### CODE REVIEW

File: ginga/gw/Desktop.py [hunk 1]
```diff
@@ -212,13 +213,13 @@ def create_toplevel_ws(self, wsname, width, height,
...
-        ## root.set_title(wsname)
-        ws = self.make_ws(wsname, wstype='tabs', use_toolbar=True)
+        # root.set_title(wsname)
+        ws = self.make_ws(wsname, wstype='tabs', use_toolbar=True)  # noqa
 
         vbox = Widgets.VBox()
         vbox.set_border_width(0)
 
-        vbox.add_widget(bnch.widget)
+        #vbox.add_widget(bnch.widget)  # TODO: Undefined bnch!
```
Comment (pllim) [inline ginga/gw/Desktop.py]: `bnch` is undefined in this method, and I can't figure out where it is supposed to come from, so I commented this.

File: ginga/gw/Desktop.py [hunk 2]
```diff
@@ -227,7 +228,7 @@ def create_toplevel_ws(self, wsname, width, height,
 
         if x is not None:
             root.move(x, y)
-        return bnch
+        #return bnch  # TODO: Undefined bnch!
```
Comment (pllim) [inline ginga/gw/Desktop.py]: `bnch` is undefined in this method, and I can't figure out where it is supposed to come from, so I commented this.

File: ginga/gw/Desktop.py [hunk 3]
```diff
@@ -244,7 +245,7 @@ def _close_page(self, ws):
         if num_children == 0:
             del self.workspace[ws.name]
             root = ws.root
-            bnch.root = None
+            #bnch.root = None  # TODO: Undefined bnch!
```
Comment (pllim) [inline ginga/gw/Desktop.py]: `bnch` is undefined in this method, and I can't figure out where it is supposed to come from, so I commented this.

File: ginga/gw/Desktop.py [hunk 4]
```diff
@@ -212,13 +213,13 @@ def create_toplevel_ws(self, wsname, width, height,
         root = self.app.make_window()
 
         # TODO: this needs to be more sophisticated
-        ## root.set_title(wsname)
-        ws = self.make_ws(wsname, wstype='tabs', use_toolbar=True)
+        # root.set_title(wsname)
+        ws = self.make_ws(wsname, wstype='tabs', use_toolbar=True)  # noqa
```
Comment (pllim) [inline ginga/gw/Desktop.py]: `ws` is defined by unused.

File: ginga/gw/Desktop.py [hunk 5]
```diff
@@ -539,11 +540,11 @@ def mypack(w):
             res = []
             for dct in cols:
                 if isinstance(dct, dict):
-                    stretch = dct.get('stretch', 0)
+                    stretch = dct.get('stretch', 0)  # noqa
```
Comment (pllim) [inline ginga/gw/Desktop.py]: `stretch` is defined by unused.

File: ginga/gw/Desktop.py (hunk 4 above)
Comment (ejeschke) [inline ginga/gw/Desktop.py]: See below

File: ginga/gw/Desktop.py (hunk 1 above)
Comment (ejeschke) [inline ginga/gw/Desktop.py]: It should be `ws`.

File: ginga/gw/Desktop.py (hunk 2 above)
Comment (ejeschke) [inline ginga/gw/Desktop.py]: It should be `ws`

File: ginga/gw/Desktop.py (hunk 3 above)
Comment (ejeschke) [inline ginga/gw/Desktop.py]: It should be `ws`

File: ginga/gw/Desktop.py (hunk 5 above)
Comment (ejeschke) [inline ginga/gw/Desktop.py]: I need to take a look at this one.  Let's leave it in but marked (as you've done) for now.

**Comment by ejeschke** [comment]
@pllim, I pushed up some fixes for the undefined `bnch`s.  Can you rebase against master?  Otherwise looks good to me.

**Comment by pllim** [comment]
@ejeschke , it is rebased.

**Comment by ejeschke** [comment]
Thanks, merging.

**Pull request merged**